.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
```

Comment:
```suggestion
dist: xenial
language: python
```

Reply:
@schenkh you could've just hit "Commit suggestion" right here in the UI ^

Reply:
Ah! Ok, thanks